src/main/java/List.java

```diff
@@ -0,0 +1,165 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class List {
```

Comment:
It is not so good to use `List` as a class name here as firstly, Java has its own `List` class which is used quite commonly so it makes it confusing which `List` you are referring to, and secondly the name does not clearly tell readers that this `List` is used to store tasks.